ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -4403,6 +4448,13 @@ interface Port-Channel100.101
    logging event link-status
    encapsulation dot1q vlan 101
    ip address 10.1.1.3/31
+   switchport tap identity 3 inner 10
```

Comment:
which device have you used for this?

```
site3-leaf1(config-s-gmu)#interface Port-Channel100.101
site3-leaf1(config-s-gmu-if-Po100.101)##interface Port-Channel100.101
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   description IFL for TENANT01
site3-leaf1(config-s-gmu-if-Po100.101)#   mtu 1500
site3-leaf1(config-s-gmu-if-Po100.101)#   logging event link-status
site3-leaf1(config-s-gmu-if-Po100.101)#   encapsulation dot1q vlan 101
site3-leaf1(config-s-gmu-if-Po100.101)#   ip address 10.1.1.3/31
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tap identity 3 inner 10
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tap mac-address dest 01:00:00:00:00:00
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tap encapsulation vxlan strip
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tool identity dot1q
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tool identity dot1q source dzgre policy
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tap truncation 120
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#   switchport tool truncation
% Invalid input
site3-leaf1(config-s-gmu-if-Po100.101)#switch?
switch  switchport

site3-leaf1(config-s-gmu-if-Po100.101)#switchport ?
  default        Configure switchport default behavior
  ethernet       Details on Ethernet 802.2 LLC header operation
  port-security  Configure MAC-address-based port security
  vlan           Details on VLAN operation
```

Reply:
ahh, it has the same issue (not applicable to sub interfaces)

Reply:
Fixed it